src/reference/asciidoc/amqp.adoc

```diff
@@ -1025,24 +1034,29 @@ When a publisher confirmation is received and correlation data is supplied, the
 The payload of the confirmation is the correlation data as defined by this expression, and the message has its 'amqp_publishConfirm' header set to `true` (`ack`) or `false` (`nack`).
 For `nack` instances, an additional header (`amqp_publishConfirmNackCause`) is provided.
 Examples: `headers['myCorrelationData']`, `payload`.
-If the expression resolves to a `Message<?>` instance (such as "`#this`"), the message emitted on the `ack`/`nack` channel is based on that message, with the additional headers added.
+If the expression resolves to a `Message<?>` instance (such as "`#this`"), the message emitted on the `ack`/`nack` channel is based on that message, with the Also see <<alternative-confirms-returns>>.
+additional headers added.
```

Comment:
I think this is a copy/paste artefact.
Probably this new `Also...` sentence should go already after `additional headers added.`